src/puppetlabs/i18n/core.clj

```diff
@@ -14,6 +14,18 @@
            (.getResources "locales.clj")
            enumeration-seq))
 
+(defn parse-info-file
+  "This function will handle the normal locales.clj file the Makefile builds in
+  a project or a list of the the locales.clj maps which we construct when we
+  build the single locales.clj that goes into an uberjar."
+  [info-file-path]
+  (let [data-structure (-> info-file-path
+                           slurp
+                           read-string)]
```

Comment:
I think my comment about using `clojure.edn/read-string` instead of `clojure.core/read-string` may have gotten lost, but still applies.